docs/docs/guides/customize-identity-model.mdx

```diff
@@ -0,0 +1,304 @@
+---
+id: customize-identity-model
+title: Customize Identity Model
+---
+
+Ory supports custom Identity Models. To learn more about the concept of Identity
+Models at Ory, please head over to
+[Identity Data Model, Custom Fields](../concepts/identity.mdx).
+
+This guide will help you create custom Identity Models as well as provide an
+example Identity Model.
+
+## Setup Custom Identity Model
+
+- Select the Ory Cloud Project, you want to change under
+  [Projects/Overview](https://console.ory.sh/projects/list).
+
+- Head over to the "Customize / General" tab (URL:
+  https://console.ory.sh/projects/$YOURPROJECTID/settings) and review the
+  currently active Identity Model.
+
+- Select a preset model or an empty template as the basis for your new custom
+  Identity Model.
+
+- Check the `Customize Identity Schema` box to enable editing on the Identity
+  Model.
+
+- Add or remove traits from the Identity Model as desired, more info in the
+  [Customize Fields](#customize-fields) section.
+
+- When finished editing name your custom Identity Model. Input the name in the
+  textbox between the checkbox and the schema.
+
+- Press `Enter` or hit the `Update` button to save it, once you have made your
+  changes. Currently, it is not possible to edit an existing Identity Model, so
```

Comment:
Maybe this needs some more details such as

```suggestion
  changes. Currently, it is not possible to edit an existing Identity Model on the same revision. The identity model can still be reused as a template for a following revision.
```

Then maybe have a section explaining what the revision would look like:

Revisions 1:
 - I create a schema with name "MyUsersv1"
 - Create the identity model and save it

Revisions 2:
 - Select the schema "MyUsersv1" as a template
 - Enter a new name (can technically be the same but there will be 2 of them then with different creation times)
 - Make the necessary adjustments to the identity model
 - Hit Save

Reply:
Great Idea! I will add such a section.